api/src/main/java/io/opentelemetry/trace/attributes/SemanticAttributes.java

```diff
@@ -125,5 +125,9 @@
   /** JDBC substring like "mysql://db.example.com:3306" */
   public static final StringAttributeSetter DB_URL = StringAttributeSetter.create("db.url");
 
+  /** Probability value used by a probability-based Span sampling strategy. */
+  public static final DoubleAttributeSetter SAMPLING_PROBABILITY =
```

Comment:
I would put this in the SDK since it is SDK specific.

Reply:
It's probably also useful for vendor API implementations though.

Reply:
Can you give an example of how an API user would want to use this attribute?

Reply:
@jkwatson this is for the moment only in java so we cannot have it here. I would suggest two things:
* Open a PR in specs to add this
* Move it as internal member in probability sampler for the moment. and document that we need to follow the specs.

Reply:
Yes, strong agree. I'll make it happen.

Reply:
@jkwatson I think an API *user* won't use this attribute (except if you use the API package at the backend, then you could use it to read the attribute back from the exported span), but *implementers* that want to be compatible with the official SDK in that regard could use it.

Reply:
I see. Do you think there's a whole class of stuff that an alternate SDK implementation would want to use from the default SDK?

Reply:
Even if that is the case, I am not sure we can have this in the API: Sampler is not part of the API (and explicitly removed from the API any "sampling" logic).

So I think this should be moved to SDK anyway.

Reply:
I was just wondering if there was going to be a need for some sort of "SDK Support" module that could contain this kind of thing. I'm not suggesting we do that now, though. :)

Reply:
https://github.com/open-telemetry/opentelemetry-specification/pull/570 for the spec change

Reply:
ok, moved into the SDK. PTAL, @bogdandrutu